mlflow/pytorch/__init__.py

```diff
@@ -325,6 +357,23 @@ def save_model(
         f.write(pickle_module.__name__)
     # Save pytorch model
     model_path = os.path.join(model_data_path, _SERIALIZED_TORCH_MODEL_FILE_NAME)
+    if artifacts:
+        if not isinstance(artifacts, dict):
+            raise TypeError("Argument artifacts should be a dict")
+
+        with TempDir() as tmp_artifacts_dir:
+            tmp_artifacts_config = {}
```

Comment:
Do we need this variable (`tmp_artifacts_config`) which doesn't seem to be used?